frontend/shared/src/components/accordion/Accordion.tsx

```diff
@@ -0,0 +1,65 @@
+import { IconAngleDown, IconAngleUp } from 'hds-react';
```

Comment:
This is 80% copy-paste of HDS accordion component. It did not support saving the state of being open or closed, also header needed to be customized.